templates/nextjs/fuel-toolchain.toml

```diff
@@ -1,6 +1,2 @@
 [toolchain]
-channel = "testnet"
-
-[components]
-forc = "0.64.0"
```

Comment:
By removing these components the `testnet` toolchain is used to its fullest, without any overriding of its components. Tested it out with @Dhaiwat10